From: =?utf-8?q?Fabian_Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Mon, 9 Mar 2026 12:08:31 +0100
Subject: d-0023-prune-wit-bindgen-blobs
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

pinned upstream to guard against the pre-built blobs changing, but we strip and
rebuild them anyway..

Gbp-Dch: Topic prune
Forwarded: no

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 compiler/rustc/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/compiler/rustc/Cargo.toml b/compiler/rustc/Cargo.toml
index 8e1e936c..4599519 100644
--- a/compiler/rustc/Cargo.toml
+++ b/compiler/rustc/Cargo.toml
@@ -24,7 +24,7 @@ rustc_public_bridge = { path = "../rustc_public_bridge" }
 # <https://github.com/rust-lang/rust/pull/136395#issuecomment-2692769062>
 [target.'cfg(target_os = "wasi")'.dependencies]
 getrandom = "=0.3.3"
-wasi = "=0.14.2"
+wasi = "=0.14.4"
 
 
 [features]
